validphys2/src/validphys/config.py

```diff
@@ -210,13 +210,16 @@ def produce_replicas(self, nreplica: int):
         """Produce a replicas array"""
         return NSList(range(1, nreplica + 1), nskey="replica")
 
+    def parse_point_prescriptions(self, point_prescriptions):
+        return NSList(point_prescriptions, nskey="point_prescription")
+
     def produce_inclusive_use_scalevar_uncertainties(
-        self, use_scalevar_uncertainties: bool = False, point_prescription: (str, None) = None
+        self, use_scalevar_uncertainties: bool = False, point_prescriptions: (list, None) = None
     ):
         """Whether to use a scale variation uncertainty theory covmat.
         Checks whether a point prescription is included in the runcard and if so
         assumes scale uncertainties are to be used."""
-        if (not use_scalevar_uncertainties) and (point_prescription is not None):
+        if not use_scalevar_uncertainties and point_prescriptions is not None:
```

Comment:
I will still address that, but I think that will involve cleaning up how the n3fit config is written since I think that can be done in a less convoluted way, but I'll have to check